scripts/sword_locales_to_json.mjs

```diff
@@ -0,0 +1,262 @@
+import fs from 'fs';
+import https from 'https';
+import { iso6393 } from 'iso-639-3';
+
+const EZRA_LOCALES_URL = 'https://raw.githubusercontent.com/ezra-bible-app/ezra-bible-app/parse-sword-languages/locales/locales.json';
+// const EZRA_LOCALES_URL = 'https://raw.githubusercontent.com/ezra-bible-app/ezra-bible-app/master/locales/locales.json';
+const SWORD_LOCALES = "locales.d/locales.conf";
+
+const extraLanguageCodes = ['cek', 'cth', 'dnj', 'esg', 'iqw', 'izz', 'ncq'];
+var indexedLangs;
+
+// Detecting Intl API support
+// https://nodejs.org/dist/latest-v14.x/docs/api/intl.html#intl_detecting_internationalization_support
+const hasFullICU = (() => {
+  try {
+    const ukrainian = new Intl.DisplayNames('uk', { type: 'language', fallback: 'none' });
+    return ukrainian.of('en') === 'англійська';
+  } catch (err) {
+    return false;
+  }
+})();
+console.log(`Running on node version ${process.version}. Full ICU support: ${hasFullICU}`);
+if (!hasFullICU) {
+  console.log(`No Intl API support detected for this node version. Script results will be limited`);
+}
+
+if (!isTesting()) {
+  indexedLangs = indexIso6393Langs();
+
+  const languages = parseSwordLocales();
+
+  const allLocales = await getEzraBibleAppLocales(EZRA_LOCALES_URL);
```

Comment:
I would prefer to keep node-sword-interface independent of Ezra Bible App. But I guess the requirement would be to maintain the supported locale list separately here, huh?

Reply:
Yes, "circle" dependencies are not good. Maybe this script should be back in Ezra Bible App. 
The other argument for this is the difference in Node versions.